dev_tools/bash_scripts_test.py

```diff
@@ -78,28 +78,31 @@ def test_pytest_changed_files_file_selection():
                        'git commit -m test --quiet --no-gpg-sign\n')
     assert result.exit_code == 0
     assert result.out == ''
-    assert result.err == ("Comparing against revision 'HEAD~1'.\n"
-                          "Found 0 differing files with associated tests.\n")
+    assert result.err.split() == (
+        "Comparing against revision 'HEAD~1'.\n"
+        "Found 0 differing files with associated tests.\n").split()
 
     result = run(script_file='check/pytest-changed-files',
                  arg='HEAD~1',
                  setup='touch file_test.py\n'
                        'git add -A\n'
                        'git commit -m test --quiet --no-gpg-sign\n')
     assert result.exit_code == 0
-    assert result.out == 'INTERCEPTED pytest file_test.py\n'
-    assert result.err == ("Comparing against revision 'HEAD~1'.\n"
-                          "Found 1 differing files with associated tests.\n")
+    assert result.out.split() == 'INTERCEPTED pytest file_test.py\n'.split()
```

Comment:
Is `split` still required here?

Reply:
No it is not - I did it for consistency. I reverted it back to keep the minimal changes required for this fix.